ChangeLog

```diff
@@ -103,24 +109,6 @@ Changes
    * Close a test gap in (D)TLS between the client side and the server side:
      test the handling of large packets and small packets on the client side
      in the same way as on the server side.
-   * Change the dtls_client and dtls_server samples to work by default over
```

Comment:
I think there was some rebase problem.
I wouldn't expect lines to be removed from the ChangeLog

Reply:
Some rebase problem. I will fix this.